client/src/message.ts

```diff
@@ -1,6 +1,24 @@
 export const ERROR = -1;
+export const OPEN = 0;
```

Comment:
Could we put these into an enum? That way you only have to import one thing to access all options.
https://www.typescriptlang.org/docs/handbook/enums.html